eng/common/Extract-ReleaseNotes.ps1

```diff
@@ -0,0 +1,47 @@
+# given a CHANGELOG.md file, extract the relevant info we need to decorate a release
+param (
```

Comment:
We should probably consider passing in the version and only returning the release notes for that given version instead of returning an entire dictionary with all of them.